### ISSUE TITLE
Iterating EUDArray pauses compilation.

### ISSUE THREAD

**Issue by armoha** [armoha/eudplib#32]
```python
from eudplib import *

def onPluginStart():
    A = EUDArray([1, 2, 3])
    for a in A:
        pass

"""
--------- Injecting plugins... ---------
Saving to output.scx... < pauses here
"""
```
Iterating EUDArray should raise `TypeError: 'EUDArray' object is not iterable`.

**Comment by armoha** [comment]
Resolved by 0e0909c978af95b61e1f2a21fd26955971ef37a4

**Comment by armoha** [comment]
0e0909c causes #34 , I think #34 is more serious than #32 so 0e0909c will reverted soon.

**Comment by armoha** [comment]
Will be fixed by 039d567
